### DISCUSSION TITLE
what are the pre-requisite to install Inbucket

### DISCUSSION THREAD

**Discussion by techsachin1** [inbucket/inbucket#366] [Q&A]
hi all,
i am first time user of inbucket. I have installed inbucket on cloud server. i have configure the as per given on site. i have to test it. i have example.xyz domain. what should i have to do to run this application properly. i have ip also. does any other dependencies required like nginx or certbot or anyhing. i have to configure for my organisation to test mail.

**Comment by jhillyerd** [comment]
Inbucket itself doesn't have per-requisites, so it's mostly going to depend on your requirements.  Running on a cloud server typically means you are exposing its HTTP and SMTP ports to the outside world, which brings some risk.  By default Inbucket will appear as an open mail relay to spammers (although it actually isn't because it does not relay), so they will start drowning you in spam.  I'd recommend setting:

INBUCKET_SMTP_DEFAULTACCEPT=false
INBUCKET_SMTP_ACCEPTDOMAINS=example.xyz

Similarly, exposing the HTTP port lets anyone browse and delete the mail.  Fronting with nginx or similar would allow you to add authentication and HTTPS.  I don't have a sample config for that, but if you search our issues, I believe one has been posted previously.

**Reply by techsachin1** [reply]
hi, thanks for your reply. can you tell e how i integrate my cloud machine to inbucket

**Comment by techsachin1** [comment]
Can you tell me how I can configure Inbucket with domain name.

**Reply by jhillyerd** [reply]
If you installed with debian or redhat package, then the config options will be part of the systemd service file.  I think you may be able to find them somewhere under /etc/systemd/system -- you are looking for this file to edit: https://github.com/inbucket/inbucket/blob/main/etc/linux/inbucket.service
